src/bridge/ui_commands.rs

```diff
@@ -74,6 +75,9 @@ impl UiCommand {
                 .expect("Focus Gained Failed"),
             UiCommand::Quit => {
                 nvim.command("qa!").await.ok(); // Ignoring result as it won't succeed since the app closed.
+            },
+            UiCommand::FileDrop(path) => {
+                nvim.command(format!("e {}", path).as_str()).await.ok();
```

Comment:
Maybe surface issue to the user if this fails? I suppose we should do the same with the other ones too...

Reply:
what about just adding `e file/path` to the command line and leave it to the user to execute it? i wonder if that's possible though

Reply:
I suspect that would be more work than folks would like. The goal is to be able to just drag a file on and get rolling. Adding another step would be less nice. I vote we just merge as is. Seems good to me